src/styles/_catalog.scss

```diff
@@ -14,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
you should specify which properties you apply the transition to. in this case all is used which is not quite ok

Reply:
I hope this solves, let's have a quick call in case I'm missing something. Thank you.